src/ukify/test/test_ukify.py

```diff
@@ -7,6 +7,7 @@
 
 import base64
 import json
+import glob
```

Comment:
sorting?

Reply:
It was sorted in my mind. But now when I run through the alphabet, I see `g` is before `j`.